README.md

```diff
@@ -43,4 +43,87 @@ If you'd like to try out the plugin pre-release, you can build it from source an
 
 ## Getting started
 
+The plugin can work in any of the  [JetBrains family of IDEs](https://www.jetbrains.com/products.html), let’s try the first example with Kubernetes and Java in IntelliJ IDE (the plugin supports both Community and Ultimate editions). Follow the installation steps above to install the plugin. Restart your IDE if prompted to activate the plugin.
+
+Clone the repository to your local machine to get your copy of the repository:
+```
+git clone https://github.com/GoogleContainerTools/google-container-tools-intellij.git
+```
+
+Open the `hello-spring-boot` example from `google-container-tools/examples` directory with your IntelliJ IDE. You can either point to the directory or to the Maven build file (`pom.xml`). The project opens and loads:
+
+<img src="docs/images/sb-hello-world-project.png" alt="opened Spring Boot hello world project" width="500"/> 
+
+This project is a very simple web application created with [the popular Spring Boot framework](https://spring.io/projects/spring-boot). It uses [Jib Maven plugin](https://github.com/GoogleContainerTools/jib) to build a container image for the project, instead of a Dockerfile. Jib is supported by the plugin and Skaffold and recommended as the best way to build optimized Java container images.
+
+Once the project loads, the plugin will detect it is preconfigured with Skaffold YAML configuration to build image and deploy to Kubernetes. A notification shows:
+
+<img src="docs/images/k8s-skaffold-notification.png" alt="Kubernetes with Skaffold notification" width="400"/> 
+
+Click `Create run configurations for Kubernetes with Skaffold` link to automatically create Kubernetes deployment and continuous development IDE run targets for the project:
+
+<img src="docs/images/k8s-skaffold-run-targets.png" alt="Kubernetes with Skaffold pre-configured run targets" width="400"/> 
+
+Now new run targets can be used to build the project and deploy it to Kubernetes or develop on Kubernetes cluster continuously. 
```

Comment:
nit: "Now new run targets" -> "Now the new run targets"

Reply:
I'm thinking we should explain what "Develop on a kuberneters cluster continuously" means, or link to the feature section somehow.

Reply:
Good suggestion. How about
```
With continuous development, the plugin uses Skaffold to watch the sources and dependencies of your project for changes, so that on any change, Skaffold builds and deploys your application to a Kubernetes cluster.
```